code/modules/power/apc.dm

```diff
@@ -380,37 +380,60 @@
 
 	if(issilicon(user) && get_dist(src,user)>1)
 		return src.attack_hand(user)
-	if (istype(W, /obj/item/weapon/crowbar) && opened)
-		if (has_electronics==1)
-			if (terminal)
-				user << "<span class='warning'>Disconnect the wires first!</span>"
+	if (istype(W, /obj/item/weapon/crowbar)) //Using crowbar
+		if (opened) // a) on open apc
+			if (has_electronics==1)
```

Comment:
no, there is 0=none;1=inserted;2=secured, we want to remove usecured electronicks with crowbar, but with secured we want to close the panel